### DISCUSSION TITLE
Can not increase watts over 45 with 7940hs on Thinkpad g16v Gen1

### DISCUSSION THREAD

**Discussion by koko1111** [JamesCJ60/Universal-x86-Tuning-Utility#170] [General]
This may be a Lenovo issue but I have seen other Lenovo laptops, "not thinkpads though" being able to tune without issue. I can not get the wattage above 45 watts, no matter what bios setting is turned off / on. Temps on stress tests do not get above 77 degrees C but watts are pegged at 45 no matter what setting is chosen from the Utility.

I am using windows 11, have Intelligent cooling and Lenovo power management in BIOS turned off . I have reinstalled windows multiple times, deleted / disabled every power management driver app or driver I could find from Lenovo but still the CPU will not go above 45 wattts.

Is this locked from Lenovo? I have ask and all they say is maybe for themal issues but at 78 C as the highest temp, that is just stupid.

**Comment by 18698120460** [comment]
samething like you. my laptop cannot overclock above 4.691ghz. And can't get wattage above 65w which should be 80 watts
